app/web/components/Select.tsx

```diff
@@ -23,6 +24,8 @@ const useStyles = makeStyles((theme) => ({
 export default function Select<T extends Record<string | number, string>>({
   id,
   className,
+  native = true,
```

Comment:
I think `native` should be false when `menuItems` is true, is that right? So when you use `<option>`, `native` should be false.

Not 100% sure though.

Reply:
I mean, the thing it's that is true by default, if I set it to false all the components that use <select> will have `native=false` which basically I don't think is retro compatible